tools/babel/messages/extract.py

```diff
@@ -20,6 +20,7 @@
 The main entry points into the extraction functionality are the functions
 `extract_from_dir` and `extract_from_file`.
 """
+from __future__ import print_function
```

Comment:
Can you also revert your changes inside tools/babel? Looks it's also third party source code.